source/MRMesh/MRConfig.cpp

```diff
@@ -222,6 +222,33 @@ bool Config::hasJsonValue( const std::string& key )
     return config_.isMember( key );
 }
 
+MRMESH_API bool MR::Config::hasEnum( Enum enumeration, const std::string& key )
+{
+    if ( !config_[key].isString() )
```

Comment:
first check if there is such member?

Reply:
Nonexistent member has a null type, getting it does not throw exceptions. Will check it again.

Reply:
Checked it. Indexing operator with inexistent key creates a null node.
The problem is, the null node can be later saved to JSON, which might clutter the output.
Fixed by making member functions `hasEnum` and `getEnum` const (same as `getColor` etc).